catroid/src/main/res/values/strings.xml

```diff
@@ -1385,7 +1385,7 @@
     <string name="formula_editor_function_log">log</string>
     <string name="formula_editor_function_log_parameter">(0)</string>
     <string name="formula_editor_function_pi">pi</string>
-    <string name="formula_editor_function_pi_parameter"></string>
```

Comment:
i guess somethings going to break due to this,
also for a pure test focused PR there should not be any changes ins string xmls